include/pg_shard.h

```diff
@@ -77,6 +77,9 @@ typedef struct DistributedPlan
 } DistributedPlan;
 
 
+#define INVALID_SHARD_ID (-1)
+typedef int64 ShardId;
```

Comment:
I agree, but we use int64 in many places :). If it's not necessary, could you change it back?